homeassistant/components/notify/telegram.py

```diff
@@ -109,7 +109,8 @@ def send_message(self, message="", **kwargs):
         # send message
         try:
             self.bot.sendMessage(chat_id=self._chat_id,
-                                 text=title + "  " + message)
```

Comment:
@balloob understood, it's just a little out of place for Telegram. I'll look at refactoring.